src/SupervisionSuccess/model/calcSavings.js

```diff
@@ -19,24 +19,34 @@ const MONTHS_IN_YEAR = 12;
  * Function that calculates one month of dollar savings due to the modeled
  * revocation reduction. This will return a negative number if there is an
  * increase in costs due to an increase in revocations.
- * @param {number} initialRevocations - revocations at the start (when month = 0)
  * @param {number} newRevocations - revocations at a given month
- * @param {{
- *   checkpoint: number
- *   savings: number
- * }[]} savingsMap - ordered list of checkpoints and associated savings
  * @param {number} marginalCostPerInmate - the marginal cost per incarcerated person
+ * @param {number} newOffensePopulation
+ * @param {number} totalCostPerInmate - the total cost per incarcerated person
+ * @param {number} numberOfFacilities - number of facilities for state
+ * @param {number} stateWideCapacity - capacity of facilities
  * @returns {number} - one month of savings
  */
-function calcSavings(initialRevocations, newRevocations, savingsMap, marginalCostPerInmate) {
-  const revocationsDiff = initialRevocations - newRevocations;
+function calcSavings({
+  newRevocations,
+  marginalCostPerInmate,
+  newOffensePopulation,
+  totalCostPerInmate,
+  numberOfFacilities,
+  stateWideCapacity,
+}) {
+  const totalPopulation = newOffensePopulation + newRevocations;
 
-  const reachedCheckPoint = savingsMap.find(({ checkpoint }) => revocationsDiff > checkpoint) || {
-    checkpoint: 0,
-    savings: 0,
-  };
-  const graduatedSavings = reachedCheckPoint.savings;
-  const marginalSavings = (revocationsDiff - reachedCheckPoint.checkpoint) * marginalCostPerInmate;
+  const facilitiesDiffRatio = (1 - totalPopulation / stateWideCapacity) * numberOfFacilities;
+
+  const facilitiesDiff = Math.floor(facilitiesDiffRatio);
+  const restPopulation = facilitiesDiffRatio % 1;
+
+  const graduatedSavings =
+    facilitiesDiff * (stateWideCapacity / numberOfFacilities) * totalCostPerInmate;
```

Comment:
Here is the difference between the excel model. Copy-pasted from the model it should look like:

`const graduatedSavings = facilitiesDiff * stateWideCapacity * totalCostPerInmate;`

But as `stateWidthCapacity` is the capacity of all state facilities (not the average capacity of a facility), I think dividing on numberOfFacilities makes sense.